actors/common/src/main/java/org/sunbird/learner/actors/InitialisationActor.java

```diff
@@ -0,0 +1,265 @@
+package org.sunbird.learner.actors;
```

Comment:
Rename to SystemInitActor.java and add to new pom project (init). Not under 'common'.